.meta.toml

```diff
@@ -2,10 +2,10 @@
 # https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [meta]
 template = "pure-python"
-commit-id = "b1a0ce9bae9fa73cb69c0d922eb73d55bd5c3080"
+commit-id = "7b24599f594f681212874ad0fa1993b6d3bc34e9"
```

Comment:
(lol that's the commit that says "zope.testing uses pure-python now", and it's not pushed anywhere, as I intended to pus it once this PR is merged.)